### DISCUSSION TITLE
zotero style使用shift +p无法调出任何可操作内容，请问怎么处理哇

### DISCUSSION THREAD

**Discussion by Cynthia0124** [MuiseDestiny/zotero-style#662] [General]
不好意思想请问一下大家，我的zotero style第一次使用shift＋p时是有具体可操作类目的，但是我设置标注颜色退出后再次打开就什么都没有了，而且标注颜色也没设置成功，disable和remove好多次也没成功，请问有什么有效的解决办法吗？谢谢大家！

**Comment by D-ruihao** [comment]
同问只有一个全文翻译的选项，我需要标签选项怎么调出来呢？

**Comment by MuiseDestiny** [comment]
禁用其他插件再试试
